src/index.tsx

```diff
@@ -120,6 +129,9 @@ const LazyHydrate: React.FunctionComponent<Props> = function(props) {
   }, [hydrated, on, ssrOnly, whenIdle, whenVisible]);
 
   if (hydrated) {
+    if (noWrapper) {
+      return <>{children}</>;
```

Comment:
I don't think a Fragment is needed here.
```suggestion
      return children;
```